### PR TITLE
MM-14943: Improve descriptions of emojis permissions

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Improve descriptions of emojis permissions

#### Ticket Link
[MM-14943](https://mattermost.atlassian.net/browse/MM-14943)

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -903,9 +903,9 @@
   "admin.permissions.permission.create_team.name": "Create teams",
   "admin.permissions.permission.create_user_access_token.description": "Create user access token",
   "admin.permissions.permission.create_user_access_token.name": "Create user access token",
-  "admin.permissions.permission.delete_emojis.description": "Delete custom emoji.",
-  "admin.permissions.permission.delete_emojis.name": "Delete Custom Emoji",
-  "admin.permissions.permission.delete_others_emojis.description": "Delete others' custom emoji.",
+  "admin.permissions.permission.delete_emojis.description": "Allow users to delete custom emoji they created.",
```
Comment (mkraft) [inline i18n/en.json]:
```suggestion
  "admin.permissions.permission.delete_emojis.description": "Allow users to delete custom emoji that they created.",
```

File: i18n/en.json [hunk 2]
```diff
@@ -903,9 +903,9 @@
...
+  "admin.permissions.permission.delete_emojis.name": "Delete Own Custom Emoji",
+  "admin.permissions.permission.delete_others_emojis.description": "Allow users to delete custom emoji created by other users.",
```
Comment (mkraft) [inline i18n/en.json]:
```suggestion
  "admin.permissions.permission.delete_others_emojis.description": "Allow users to delete custom emoji that were created by other users.",
```

File: i18n/en.json (hunk 1 above)
Comment (jespino) [reply i18n/en.json]: @wiersgallak ?

File: i18n/en.json (hunk 2 above)
Comment (jespino) [reply i18n/en.json]: @wiersgallak ?

Comment (wiersgallak) [reply i18n/en.json]: +1 Looks good.

**Pull request merged**